scripts/buildCss.js

```diff
@@ -0,0 +1,41 @@
+// core packages
+const fs = require('fs'),
+      path = require('path')
+
+// npm packages
+const simpleIcons = require('simple-icons')
+
+
+const basePath = path.join(__dirname, '..')
+
+const attributedIcons = Object.values(simpleIcons).map(icon => {
+	return {
+		color: icon.hex,
+		cssClass: icon.title
+			.toLowerCase()
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
```

Comment:
This (comment) feels outdated since this is about CSS classes (also goes for [buildTestPage:18](https://github.com/metaa/simple-icons-font/blob/d5bc9f5dbbd38a17ecbc48b327cee5bfe8d3a937/scripts/buildTestpage.js#L18))